components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -133,3 +133,17 @@ fun String.isSameOriginAs(other: String): Boolean {
     }
     return canonicalizeOrigin(this) == canonicalizeOrigin(other)
 }
+
+/**
+ * Truncates this [String] to the provided size.
+ *
+ * @param size to truncate to.
+ * @throws StringIndexOutOfBoundsException in case size is < 0.
+ */
+fun String.truncate(size: Int): String {
```

Comment:
Was looking around for an existing Kotlin function. They have so many helpers that I was wondering if they had one for that too. `take()` looks like it could be it?
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/take.html

Reply:
Oh, nice. Let me try and remove our own then :).

Reply:
Updated to switch to `take`.